index.html

```diff
@@ -1,31 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8"> 
-    <link rel="stylesheet" href="assets/ol/ol.css" type="text/css">
-    <style>
-      .map {
-        height: 400px;
-        width: 100%;
-      }
-      #inputwkt {
-        width: 100%;
-      }
-    </style>
-    <script src="assets/ol/ol.js"></script>
-    <script src="assets/proj4/proj4.js"></script>
-    <script src="assets/jquery/jquery-3.3.1.min.js"></script>
-    <title>sitnLayers</title>
-  </head>
-  <body>
-    <h2>Exemple 1</h2>
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="assets/ol/ol.css" type="text/css">
+  <link rel="stylesheet" href="assets/ol-ext/ol-ext.min.css" type="text/css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
```

Comment:
Maybe it would be good to locally download all the needed packages

Reply:
I agree! But bootstrap is only required for the demo, it is not used in the plugin.